Files/BaseLayout.cs

```diff
@@ -445,7 +445,7 @@ protected override async void OnNavigatedTo(NavigationEventArgs eventArgs)
             }
             else
             {
-                ParentShellPageInstance.NavToolbarViewModel.CanRefresh = false;
+                ParentShellPageInstance.NavToolbarViewModel.CanRefresh = true;
```

Comment:
For some reason it appears I can only refresh once, after that the button gets disabled.

Reply:
Thanks, should be fixed now